packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -54,16 +61,34 @@ export function trackInteractionToNextPaint(
       }
     }
 
-    const newInteraction = longestInteractions.estimateP98Interaction()
-    if (newInteraction) {
-      interactionToNextPaint = newInteraction.duration
+    const inpInteraction = longestInteractions.estimateP98Interaction()
+    if (inpInteraction) {
+      interactionToNextPaint = inpInteraction.duration
+      if (!telemetryCollected) {
+        telemetryCollected = true
+        addTelemetryDebug('INP outlier', {
+          inp: interactionToNextPaint,
+          interaction: inpInteraction
+            ? {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏`inpInteraction` is always truthy here, you could remove this ternary